packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts

```diff
@@ -192,7 +192,7 @@ export async function createDeclarationCreator(
       let output = service.getEmitOutput(filename, true, true);
       return {
         name: output.outputFiles[0].name.replace(
-          dirname,
+          normalizePath(dirname),
```

Comment:
this couldn't be replaced correctly because the `.name` was already normalized the replacing pattern was also normalized, but the input string (`dirname`) was not